hetmech/test_degree_weight.py

```diff
@@ -32,5 +27,10 @@ def test_disease_gene_example_dwwc():
     disease_index = get_node_to_position(graph, 'Disease')
     i = gene_index[graph.node_dict['Gene', 'IRF1']]
     j = disease_index[graph.node_dict['Disease', 'Multiple Sclerosis']]
+
+    # Warning: the WC (walk count) and PC (path count) are only equivalent
+    # because none of the GiGaD paths contain duplicate nodes. Since, GiGaD
+    # contains duplicate metanodes, WC and PC are not gauranteed to be the
+    # same. However, they happen to be equivalent for this example.
```

Comment:
I can't tell from this comment whether the metapath GiGaD contains duplicate nodes (line 33 seems to say yes) or not (line 32 seems to say no).